hca-foundation/juniorachievement
# Tb-Containerization

# Description
**New**
- Containerized frontend and backend apps using Docker
- Added frontend app documentation to ReadME
- Alphabetized school and district dropdown menus
- Added a 'date' field to capture date assessments submitted

**Fix**
- Updated RemoteURL to the deployed server in DataManager
- Updated CORS in backend app settings.py to allow communication between apps
- Corrected typos throughout assessments
- Updated and reordered fields in serializer

**Refactor**
- Changed Assessment model's Q22-35 to IntFields and updated ViewSet
- Restructured directories and files to more clearly indicate frontend and backend apps

**Style**
N/A

# Testing Instructions
- Submit an assessment and check that the 'date' field exists and today's date is the value
- Ensure answers to Q22-35 are integers, not strings
- Check that schools and districts listed in dropdowns are alphabetized
- Test frontend app's documentation in ReadME
